README.md

```diff
@@ -117,6 +115,10 @@ You can deploy this project:
 - Twitter: [@twitterhandle](https://twitter.com/Deby_grey)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/deborah-fashoro)
 
+👤 **Martin Karugaba**
+
+- GitHub: [@githubhandle](https://github.com/martinkarugaba)
+
```

Comment:
Kindly add my linkedin and my twitter to here.
[My linkedin profile](https://www.linkedin.com/in/martinkarugaba/)
[My twitter profile](https://twitter.com/martin_karugaba)